app/controllers/health_checks_controller.rb

```diff
@@ -8,9 +8,22 @@ class HealthChecksController < ActionController::Base
   newrelic_ignore_apdex
 
   def show
+    migrations = check_migrations
     body = {
       healthy: true
-    }.merge(Rails.application.config.build_version || {})
+    }.merge(Rails.application.config.build_version || {}).merge(migrations)
     render(json: body, status: :ok)
   end
+
+  private
+
+  def check_migrations
+    migrations = []
+    pending_migrations = false
+    ActiveRecord::Base.connection.migration_context.migrations_status.each do |status, version, name|
+      migrations << { status: status, version: version, name: name }
+      pending_migrations = true if status != "up"
+    end
+    { migrations: migrations, pending_migrations: pending_migrations }
```

Comment:
Move `pending_migrations` closer to the top of the page since `migrations` is a long list.
```suggestion
    { pending_migrations: pending_migrations, migrations: migrations }
```